crates/lovely-core/src/sys.rs

```diff
@@ -110,6 +127,7 @@ pub static lua_isstring: Lazy<Symbol<unsafe extern "C" fn(*mut LuaState, isize)
 /// # Safety
 /// Makes a lot of FFI calls, mutates internal C lua state.
 pub unsafe fn load_module<F: Fn(*mut LuaState, *const u8, isize, *const u8) -> u32>(state: *mut LuaState, name: &str, buffer: &str, lual_loadbuffer: &F) {
+    println!("Loading module {name}");
```

Comment:
Remove this print statement or use one of the log macros - `info!()` seems to be the best fit.